crates/sui-indexer/src/apis/read_api.rs

```diff
@@ -173,13 +182,27 @@ impl ReadApiServer for ReadApi {
     }
 
     async fn get_latest_checkpoint_sequence_number(&self) -> RpcResult<CheckpointSequenceNumber> {
+        if self
+            .method_to_be_forwarded
+            .contains(&"get_latest_checkpoint_sequence_number".to_string())
```

Comment:
nit: Use a constants file? 

question: why do sometimes they have sui in front of the method headers?

Reply:
assuming you meant a constant for the string literal of the function name of  `get_latest_checkpoint_sequence_number`.
The function name will be set in config like yaml file and only checked once in each function, that's why I felt having a separate constant var would not help.